tools/rvn/CommandLineApp.cs

```diff
@@ -731,17 +637,14 @@ private static void ValidateSetupOptions(CreateSetupPackageParameters parameters
 
                 case LetsEncrypt: return;
 
+                case Unsecured: return;
+                
                 case OwnCertificate: return;
                 
-                default: throw new InvalidOperationException($"{parameters.Mode} mode is invalid{Environment.NewLine}-m|--mode option must be set. Please use either '{OwnCertificate}' or '{LetsEncrypt}'");
+                default: throw new InvalidOperationException($"{parameters.Mode} mode is invalid. -m|--mode option must be set.{Environment.NewLine}Please use either '{Unsecured}', '{OwnCertificate}' and '{LetsEncrypt}'");
```

Comment:
part of this message is very similar to 541. Could be extracted to a common place.

Reply:
There's a completely different message in line 541.